napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -335,3 +343,31 @@ def _on_change(self, event=None):
                 getattr(self._target, self._attribute) == self._onstate
             ):
                 self.toggle()
+
+
+def open_perspective_popup(btns: QtViewerButtons):
```

Comment:
Why not make this an instance method @tlambert03?

Reply:
Thoughts @tlambert03? I don't have a preference.

Reply:
moved